src/server/utils/query/bigquery.ts

```diff
@@ -0,0 +1,286 @@
+import { parse } from "path";
```

Comment:
this file is mostly AI generated version that yield super broken SQL but demonstrates something hat may look kind of close to the end goal